modules/serverless-enabling-pvc-support.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: REFERENCE
+[id="serverless-enabling-pvc-support_{context}"]
+= Enabling PVC support
+
+Some serverless applications need permanent data storage. To achieve this, you can configure Persistent Volume Claims (PVCs) for your Knative services.
+
+* The `"kubernetes.podspec-persistent-volume-claim"` extension controls whether Persistent Volumes (PVs) can be used with Knative Serving.
+* The `"kubernetes.podspec-persistent-volume-write"` extension controls whether PVs are available to Knative Serving with the write access.
+
+To enable Knative Serving to use PVCs and write to them, modify the `KnativeServing` custom resource (CR) to include the following YAML:
+
+.Enabling PVCs with write access
+[source,yaml]
+----
+...
+spec:
+  config:
+    features:
+      "kubernetes.podspec-persistent-volume-claim": enabled
+      "kubernetes.podspec-persistent-volume-write": enabled
+...
+----
+
+To claim a PV, modify your service to include the PV configuration. For example, the following YAML claims a PV with write access:
+
+.PersistentVolumeClaim configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: example-pv-claim
+  namespace: my-ns
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+----
+
+.Knative service PVC configuration
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
```

Comment:
Add the namespace here to make clear that pvc and service need to have the same value?